src/db.ts

```diff
@@ -18,13 +18,25 @@ let instance: DB;
 export class Captcha extends Model {
     public id!: string;
     public user_id!: string;
+    public quiz_id!: string;
+    public text!: string;
```

Comment:
Why are we including text inside the captcha?

Reply:
The text is the question, so we can generate it unique at the start of a quiz and transmit it later.

Reply:
If the purpose of the field is to make sure another captcha is not made with the same settings then it seems pretty inelegant to store it all instead of just a hash.

Maybe I'm misunderstanding it's purpose.

Reply:
If you look at main.ts:87 we generate the unique captcha via `get_unique_captchas` and the store the question itself (you are attacking X, what is Y). We generate the question in it's entirety once and then can retrieve it later along with the corresponding answer.